src/utilities/codemirror/custom-folder.test.ts

```diff
@@ -345,3 +360,42 @@ describe('foldRequest', () => {
     });
   });
 });
+
+describe('foldVariables', () => {
+  test('Parameters should return the correct from and to', () => {
+    PARAMETERS.forEach(parameter => {
+      const doc = `${parameter}`;
+      const tree = parser.parse(doc);
+      const state = EditorState.create({ doc });
+
+      const node = tree.topNode.getChild('ParameterDeclaration');
+      if (!node) {
+        throw new Error('node not found');
+      } else {
+        expect(node).not.toBeNull();
+        expect(foldVariables(node, state)).toStrictEqual({
+          from: '@INPUT_PARAMS_BEGIN'.length,
+          to: doc.length,
+        });
+      }
+    });
+  });
+  test('Parameters should return the correct from and to', () => {
+    LOCALS.forEach(local => {
+      const doc = `${local}`;
```

Comment:
the template literal is extraneous, local is a string an assignment will suffice

Reply:
Addressed with commit 3244803be50f2ee2de696c1459c4dc03d3d4bb87